docs/users/cli-usage.md

```diff
@@ -90,7 +90,7 @@ composer create-project drupal-composer/drupal-project:8.x-dev my-drupal8-site -
 cd my-drupal8-site
 ```
 
-_You can find more information on composer and how to use it [here](https://github.com/drupal-composer/drupal-project)._
+_Find more information on composer and how to use it [here](https://github.com/drupal-composer/drupal-project)._
```

Comment:
"More information about composer is at ..." ???

Reply:
I think the imperative makes sense here, and in general, for *See the* and *Find more*, etc.

Turns out the link text is wrong here - shouldn't link to [here](#). 

Changing to:

_Find more information [on composer and how to use it](https://github.com/drupal-composer/drupal-project)._